build_web_compilers/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: build_web_compilers
-version: 0.4.3
+version: 0.4.4-dev
```

Comment:
Don't we need to update the minimum SDK constraint to require `2.1.0-dev..??` for when this was added? It's not in dart2 stable

Reply:
Good catch!